src/state/process.rs

```diff
@@ -735,8 +720,15 @@ where
             return Ok(());
```

Comment:
remove if !self.is_leader filter condition. A node should process any vote which pass above filter condition by its round.

Reply:
It seems useless. If self is a leader, receive a future signed vote is impossible.

Reply:
yes, ignore it